text/0035-verifiable-burning.md

```diff
@@ -0,0 +1,165 @@
+- Feature Name: Verifiable Burning
+- Start Date: 2022-04-07
+- MCIP PR: [mobilecoinfoundation/mcips#0035](https://github.com/mobilecoinfoundation/mcips/pull/0035)
+- Tracking Issue: https://github.com/mobilecoinfoundation/mobilecoin/issues/1779
+
+# Summary
+[summary]: #summary
+
+Create a way to provably burn funds which can be audited trustlessly by anyone.
+
+# Motivation
+[motivation]: #motivation
+
+Burning means destroying funds (or, rendering them unspendable).
+
+There are a few reasons this may be done:
+* To adjust the supply of a synthetic asset
+* To create a deflationary pressure on a token
+
+Particularly, [MCIP #31](https://github.com/mobilecoinfoundation/mcips/pull/0031)
+will provide the foundations for synthetic assets in MobileCoin.
+
+It is quite easy to make funds unspendable by just sending them to a random public address.
+If the public address is generated by hashing to curve, then it is infeasible for anyone
+to find the corresponding private keys, so no one can find and spend this money.
+
+However, in applications, we generally want burning to be transparent and auditable,
+so that anyone can easily verify when and how much funds were burned in this way.
+
+This purpose is served if the view private key of the burn address is known to everyone,
+so that they can view key scan to see the provably burned funds (and read the [MCIP #3](https://github.com/mobilecoinfoundation/mcips/pull/0003) memos
+on the burned funds, which may also be useful.)
+
+Therefore, we construct and standardize a burn address such that the view private key
+is a known constant, and the spend private keys are unknowable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+This proposal creates a standard burn address, such that the view private key is known.
+
+To provably burn funds, simply send them to the burn address. You may attach a memo if this serves your purposes.
+
+To verify that burning occurred, use the view private key to the burn address and view-key scan the entire blockchain.
+Every provably burning event will be revealed, and you can see the block index, the amount, and read the memo field.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The burn address construction and rationale is as follows:
```

Comment:
What prevents *anyone* from burning arbitrarily via sending funds to the burn address?

Reply:
Nothing, anyone is welcome to burn to the burn address. If some sort of authentication e.g. for integration with a redemption workflow, then I'd want that to be like a Mac value in the encrypted memo or something like this